views/partials/layout.njk

```diff
@@ -49,7 +49,7 @@
     integrity="sha256-VazP97ZCwtekAsvgPBSUwPFKdrwD3unUfSGVYrahUqU="
     crossorigin="anonymous"></script>
 
-  {% if feComponents.jsInclude %}
+  {% if feComponents.jsIncludes %}
```

Comment:
Need to correct this typo in the instructions too (and I just checked... it's there in the profile). Annoying that no linting has picked this up!